src/AnswersHeadlessContext.ts

```diff
@@ -1,4 +1,4 @@
-import { AnswersHeadless } from '@yext/answers-headless';
+import { AnswersHeadless } from '@yext/answers-headless-react';
```

Comment:
should these imports from the top level src folder be importing from its own package? or can it stay as '@yext/answers-headless'

Reply:
my bad I fixed these locally but didn't add them before comitting